docs/features/feature-guides/compliance-forms/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Compliance Forms
+<FeatureAvailability/>
+
+**DataHub Compliance Forms** simplify the process of documenting, annotating, and classifying your data assets while ensuring accountability by assigning tasks to the appropriate data owners, stewards, and subject matter experts.
+
+With flexible configurations, Compliance Forms empower teams to ask the right questions to the right people for a targeted set of assets, streamlining compliance and governance workflows.
+
+## What are DataHub Compliance Forms?
+
+Think of **Compliance Forms** as structured surveys for your data assets: a series of questions that must be answered to ensure an asset meets your documentation and classification standards.
+
+Key Features:
+
+* **Configurable Questions:** Tailor Forms to collect exactly the information you need.
```

Comment:
At this point - it's still quite abstract what type of questions we might have here. Or the fact that questions imply metadata collection. Thinking of ways to communicate this. My take is above